submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sSubmitter.java

```diff
@@ -534,6 +535,49 @@ public ServeResponse deleteServe(ServeRequest spec)
     }
   }
 
+  @Override
+  public ServeResponse checkServePodReady(String name)
+      throws SubmarineRuntimeException {
+    V1Pod pod = null;
+    V1PodList list;
+    try {
+      list = coreApi.listPodForAllNamespaces(null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null);
+    } catch (ApiException e){
+      throw new SubmarineRuntimeException(e.getCode(), "K8s ApiException" + e.getMessage());
+    }
+    for (V1Pod item : list.getItems()){
+      try {
+        if (item.getMetadata().getLabels().get("app").equals(name)){
+          pod = item;
+          break;
+        }
+      } catch (NullPointerException e){
+        continue;
+      }
+    }
+    if (pod == null) {
+      throw new SubmarineRuntimeException(404, "Pod " + name + " not fund.");
```

Comment:
The error code should be replaced with constant

Reply:
But some functions above is not using constant error code.